bin/main.rb

```diff
@@ -1,2 +1,60 @@
 #!/usr/bin/env ruby
-puts 'Hello World'
+
+puts 'Welcome to Tic Tac Toe!'
+print 'Enter player 1 name: '
+player_one = gets.chomp
+
+print 'Good. Enter player 2 name: '
+player_two = gets.chomp
+
+puts "\nPlayer 1 is #{player_one} using symbol X. \nPlayer 2 is #{player_two} using symbol O. \n\nLet's Play!"
+
+# A game board will be displayed to the user, consisting of numbers from 1 to 9
+
+# We also have arrays that stores the values of selected moves by the players
+p1_move = []
+p2_move = []
+
+puts "\nWith each boxes displayed from 1 to 9, each player can type in a value >=1 and <= 9"
+
+count = 0
+while count < 9
```

Comment:
As previous TSE specified you need to use a variable to terminate the loop not a number.for example
````ruby

game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
`````

Reply:
Fixed.